logging/bin/getlogs.py

```diff
@@ -95,6 +103,21 @@ def validate_input(dict):
             print("Please remove single quotes ('') from search argument.")
             sys.exit()
 
+def open_port(dict):
```

Comment:
I know some of the has already done but I want to point out that I'm not a fan of passing around dictionaries to functions like this. I think arguments should be passed explicitly. You could define this function with the arguments such as:
`def open_port(host, port)`
and then pass the dictionary, which still isn't great but like this:

`open_port(**dict)`
That should resolve the dictionary to the arguments of the function.

The reason I prefer explicit arguments is that it makes it much easier to maintain and readable.